Jenkinsfile

```diff
@@ -1,101 +1,8 @@
 #!/usr/bin/env groovy
 
-REPOSITORY = 'policy-publisher'
-DEFAULT_SCHEMA_BRANCH = 'deployed-to-production'
-
 node {
   def govuk = load '/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy'
-
-  properties([
-    buildDiscarder(
-      logRotator(
-        numToKeepStr: '50')
-      ),
-    [$class: 'RebuildSettings', autoRebuild: false, rebuildDisabled: false],
-    [$class: 'ThrottleJobProperty',
-      categories: [],
-      limitOneJobWithMatchingParams: true,
-      maxConcurrentPerNode: 1,
-      maxConcurrentTotal: 0,
-      paramsToUseForLimit: 'policy-publisher',
-      throttleEnabled: true,
-      throttleOption: 'category'],
-    [$class: 'ParametersDefinitionProperty',
-      parameterDefinitions: [
-        [$class: 'BooleanParameterDefinition',
-          name: 'IS_SCHEMA_TEST',
-          defaultValue: false,
-          description: 'Identifies whether this build is being triggered to test a change to the content schemas'],
-        [$class: 'StringParameterDefinition',
-          name: 'SCHEMA_BRANCH',
-          defaultValue: DEFAULT_SCHEMA_BRANCH,
-          description: 'The branch of govuk-content-schemas to test against']]
-    ],
-  ])
-
-  try {
-    govuk.initializeParameters([
-      'IS_SCHEMA_TEST': 'false',
-      'SCHEMA_BRANCH': DEFAULT_SCHEMA_BRANCH,
-    ])
-
-    if (!govuk.isAllowedBranchBuild(env.BRANCH_NAME)) {
-      return
-    }
-
-    stage("Checkout") {
-      checkout scm
-    }
-
-    stage("Clean up workspace") {
-      govuk.cleanupGit()
-    }
-
-    stage("git merge") {
-      govuk.mergeMasterBranch()
-    }
-
-    stage("Configure Rails environment") {
-      govuk.setEnvar("RAILS_ENV", "test")
-    }
-
-    stage("Set up content schema dependency") {
-      govuk.contentSchemaDependency(env.SCHEMA_BRANCH)
-      govuk.setEnvar("GOVUK_CONTENT_SCHEMAS_PATH", "tmp/govuk-content-schemas")
-    }
-
-    stage("bundle install") {
-      govuk.bundleApp()
-    }
-
-    stage("rubylinter") {
-      govuk.rubyLinter()
-    }
-
-    stage("Set up the DB") {
-      govuk.runRakeTask("db:drop db:create db:schema:load")
-    }
-
-    stage("Precompile assets") {
-      govuk.precompileAssets()
-    }
-
-    stage("Run tests") {
-      govuk.runRakeTask("default")
-    }
-
-    stage("Push release tag") {
-      govuk.pushTag(REPOSITORY, env.BRANCH_NAME, 'release_' + env.BUILD_NUMBER)
-    }
-
-    govuk.deployIntegration(REPOSITORY, env.BRANCH_NAME, 'release', 'deploy')
-
-  } catch (e) {
-    currentBuild.result = "FAILED"
-    step([$class: 'Mailer',
-          notifyEveryUnstableBuild: true,
-          recipients: 'govuk-ci-notifications@digital.cabinet-office.gov.uk',
-          sendToIndividuals: true])
-    throw e
-  }
+  govuk.buildProject(
+    false // disable SASS linting
```

Comment:
Yeah, that would've been better probably. Although it might make it easier to add arguments, which I'd like to avoid :) I'm hoping that I can figure out a way to only lint scss diffs, so that we can remove the argument altogether.